### ISSUE TITLE
ebs snapshot action broken: 'EC2' object has no attribute 'create_snapshots'

### ISSUE THREAD

**Issue by nitrocode** [cloud-custodian/cloud-custodian#4740]
TLDR: @kapilt  I believe the function `create_snapshots` is missing from boto3 in spite of the docs

error in cloudwatch
```
 'EC2' object has no attribute 'create_snapshots'
```

* change was put in on Jun 13 2019 which is after [0.8.43.1 and before 0.8.44.0 release](https://pypi.org/project/c7n/#history) so to fix this we pinned to `0.8.43.1`
* [source currently](https://github.com/cloud-custodian/cloud-custodian/blame/5f12e810344e62b4756bf9d7b94888e61a1ff35d/c7n/resources/ec2.py#L1363) 
* [source before change](https://github.com/cloud-custodian/cloud-custodian/blame/ec4981213123c28d7e59cc53e121761b281d0a56/c7n/resources/ec2.py#L1347)
* [I'm not sure how this test passes right now tho](https://github.com/cloud-custodian/cloud-custodian/blame/5f12e810344e62b4756bf9d7b94888e61a1ff35d/tests/test_ec2.py#L1012)
* also create_snapshots does exist in the [boto3 documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.create_snapshots) but not in the [boto3 code itself](https://github.com/boto/boto3/search?q=create_snapshots&unscoped_q=create_snapshots).

**Comment by kapilt** [comment]
the issue is your probably deploying in lambda and need to include a more recent version of boto3, botocore .. on mode use `packages: [boto3, botocore, urllib3]`

if your doing this via virtualenv install you'll need to just update your boto3 ` pip install -U boto3`

**Comment by nitrocode** [comment]
do I need to do this all the time? 

```
packages: [boto3, botocore, urllib3]
```
